lib/http_requests.py

```diff
@@ -143,15 +144,23 @@ def get_advisories(branch_name):
         if version in seen:
             continue
 
+        assembly_data = yml_data[version].get('assembly', {})
+        assembly_type = assembly_data.get('type', '').lower()
+
+        # Skip 'custom' assembly types
+        if assembly_type == 'custom':
+            continue
+
         depth = 0
         current_version = version
         current_advisories, jira_link = process_version_advisories(current_version, yml_data, seen)
         if not current_advisories:
             current_advisories = {}
 
+        # Process advisories for basis versions
         while depth < MAX_DEPTH:
             depth += 1
-            basis_version = yml_data[current_version].get('assembly', {}).get('basis', {}).get('assembly')
+            basis_version = assembly_data.get('basis', {}).get('assembly')
```

Comment:
Hmm shouldn't it be `yml_data[current_version]` here still?

Reply:
You are correct. The assembly_data in the loop should refer to yml_data[current_version]. The assembly_data should be updated inside the loop to get the assembly details of the basis version being processed.